docs/en/sql-reference/functions/distance-functions.md

```diff
@@ -503,3 +503,17 @@ Result:
 │           0.007722123286332261 │
 └────────────────────────────────┘
 ```
+
```

Comment:
There are also no examples using Array for L1/L2/Linf/LpDistance() and the example is trivial enough anyways. Should be okay to omit l. 506ff for consistency reasons.